MekHQ/src/mekhq/gui/view/MissionViewPanel.java

```diff
@@ -1006,5 +1006,14 @@ public void mouseClicked(MouseEvent e) {
         gridBagConstraints.fill = GridBagConstraints.BOTH;
         gridBagConstraints.anchor = GridBagConstraints.NORTHWEST;
         pnlStats.add(txtDesc, gridBagConstraints);
+        
+        // for StratCon, contract score is irrelevant and only leads to confusion, so we
```

Comment:
I'd prefer this to be located alongside the lblScore and txtScore initialization, preferrably just not initializing them in this case to prevent oddities if we end up adding further labels to this panel.